VideoLocker/res/values/errors.xml

```diff
@@ -63,6 +63,19 @@
     <!-- Alert dialog content shown when user attempts to view a video that hasn't been synced -->
     <string name="video_offline_message">Please select a video that you’ve downloaded onto your device.</string>
 
+    <!-- New version available -->
+    <!-- Snackbar notification shown when the current version of the app is unsupported -->
+    <string name="app_version_unsupported">Your version of the app is no longer supported. Upgrade to the latest version.</string>
+    <!-- Snackbar notification shown when the current version of the app is deprecated -->
+    <string name="app_version_deprecated">Your version of the app is no longer supported. Upgrade to the latest version by {last_supported_date} to continue accessing course content online.</string>
+    <!-- Snackbar notification shown when the current version of the app is outdated -->
+    <string name="app_version_outdated">A new version of the app is available. Upgrade to the latest version.</string>
+    <!-- Snackbar action button label for opening an app store to upgrade the app to the latest version -->
+    <string name="app_version_upgrade_button">Upgrade</string>
+    <!-- Toast notification for when the user presses the upgrade action button from the Snackbar
+         notifications while no app store or browser is installed on the device -->
+    <string name="app_version_upgrade_app_store_unavailable">Play Store not found on device.</string>
```

Comment:
@catong: Note that I have now changed this to "No supported app store found on device.", since the actual app store we're directing to has been [made][1] configurable (and would not necessarily always be the Play Store). Can you confirm this phrasing, or do you want to suggest some edits to it?

[1]: https://github.com/edx/edx-app-android/pull/748#discussion_r73464816